### ISSUE TITLE
Efforts of testing WCAG 3.0 [Deque feedback]

### ISSUE THREAD

**Issue by melaniephilipp** [w3c/silver#369]
WCAG 3.0 is looking to in a variety of areas add greater granularity and complexity to accessibility testing such as: 0-4 scoring, averaging across functional categories, counting elements to calculate percentages per Outcome, etc. This has the potential to significantly increase the cost of testing. To avoid this, Deque wants to see AGWG set a goal of how much the effort involved in testing WCAG 3 will be compared to testing WCAG 2.1.

Organisations have a finite budget for their accessibility work. If the cost of auditing goes up, the frequency with which an organization does their testing will go down. This increases the time between accessibility issues making it onto a web page, and it getting discovered and resolved. All other things being equal, simply having greater granularity in WCAG 3 can result in a reduction in accessibility on the web. This could also result in organisations not moving to WCAG 3.0 at all, as WCAG 2.1 might prove more cost-effective.

To avoid this problem, we want to encourage AGWG to set itself a goal of how much (on average) the effort involved in a WCAG 3.0 audit can increase from WCAG 2.2 at an equivalent level. Our recommendation would be to try and keep the level of effort roughly equivalent. When increasing the efforts involved in certain areas, that effort should be offset by simplifying, or allowing greater automation in other areas.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by alastc** [comment]
Today the group decided that:

> Don't add to requirements document but schedule a conversation about the holistic cost of WCAG 3 after we make more progress (add to WCAG 3 schedule after conformance decisions for future discussion)

We will close the issue now, but the schedule will be updated to include this aspect.
